src/test/resources/minio-bucket-creation-commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+/usr/bin/mc config host add myminio http://$1 BQA2GEXO711FVBVXDWKM uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC;
+/usr/bin/mc mb myminio/insights-upload-perma;
+/usr/bin/mc mb myminio/insights-upload-rejected;
+/usr/bin/mc policy download myminio/insights-upload-perma;
```

Comment:
it seem that locally this command didn't work for me since in the log it prints the usage output:
```
16:31:12.000 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT: Added `myminio` successfully.
16:31:12.000 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT: Bucket created successfully `myminio/insights-upload-perma`.
16:31:12.000 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT: Bucket created successfully `myminio/insights-upload-rejected`.
16:31:12.000 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT: Name:
16:31:12.000 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy - manage anonymous access to buckets and objects
16:31:12.076 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT: USAGE:
16:31:12.077 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy set [FLAGS] PERMISSION TARGET
16:31:12.077 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy set-json [FLAGS] FILE TARGET
16:31:12.077 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy get [FLAGS] TARGET
16:31:12.077 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy get-json [FLAGS] TARGET
16:31:12.077 [tc-okhttp-stream-2142129482] INFO  o.j.xavier.integrations.EndToEndTest - [MINIO-MC-LOG] STDOUT:   mc policy list [FLAGS] TARGET
```